warp/_src/generated_files.py

```
@@ -0,0 +1,111 @@
+"""
+            f.write(copyright_notice)
+            export_builtins(f)
+
+        print(f"Finished writing {export_path}")
+    except FileNotFoundError:
+        print(f"Error: The file '{export_path}' was not found.")
+    except PermissionError:
+        print(f"Error: Permission denied. Unable to write to '{export_path}'.")
+    except OSError as e:
+        print(f"Error: An OS-related error occurred: {e}")
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)